Firestore/core/test/firebase/firestore/util/executor_std_test.cc

```diff
@@ -212,11 +211,16 @@ TEST_F(ScheduleTest, PopBlockingIsNotAffectedByIrrelevantRemovals) {
     EXPECT_EQ(schedule.PopBlocking(), 1);
   });
 
-  while (schedule.empty()) {
+  // Wait (with timeout) for both values to appear in the schedule.
```

Comment:
It's possible for this wait to succeed when only one of two values has been pushed to the queue.